### PR TITLE
SDIT-1835 Ignore updates to adjustments deleted

### PR DESCRIPTION
**Pull request by andymarke**
It is possible to process an UPSERT event from NOMIS after an adjustment is deleted - this can happen if events are delayed. In this scenario just ignore the update rather than sending to DLQ

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/sentencing/SentencingAdjustmentsMigrationService.kt [hunk 1]
```diff
@@ -77,8 +77,9 @@ class SentencingAdjustmentsMigrationService(
             nomisApiService.getKeyDateAdjustment(nomisAdjustmentId)
           }
 
+        // this service needs deleting now migration is done
```
Comment (andymarke) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/sentencing/SentencingAdjustmentsMigrationService.kt:80]: will do this in a separate PR

**Review by mikehalmamoj — APPROVED** [review]
